lib/linguist/languages.yml

```diff
@@ -34,7 +34,6 @@
 # test changes in `test/test_blob.rb`.
 #
 # Please keep this list alphabetized. Capitalization comes before lowercase.
----
```

Comment:
Why are you removing this? Please don't.

Reply:
Apologies for that, though it seems either my Prettier or another formatter extension was causing that on save, I'll add that back in my next few sets of commits.

Reply:
Resolved the codeblock issue [here](https://github.com/github-linguist/linguist/pull/6612/commits/428f5d21f1f95205d50e9e8f73aa009c4c078640).